frontend/src/Routes/routes.tsx

```diff
@@ -20,9 +18,7 @@ const Main = () => {
       <Route path="/admin-login" element={<AdminLogin />} />
       <Route path="/admin-register" element={<AdminSignin />} />
       <Route path="/dashboard" element={<Dashboard />} />
-      <Route path="/admin/category/add" element={<AddCategoryPage />} />
-      <Route path="/admin/categories" element={<CategoryListPage />} />
-      
+      <Route path="/admin/category/{category_id}/add" element={<AddWordPage />} />
```

Comment:
Do not hard code the **{category_id}** word. What I mean is the **{category_id}**  should be dynamic . For example : `/admin/category/1/add ,`     
`/admin/category/2/add` , etc.